packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -782,26 +782,12 @@ addFieldsDict(Posts, {
     viewableBy: ['guests'],
     insertableBy: ['members'],
     editableBy: [userOwns, 'sunshineRegiment', 'admins'],
-    label: "Facebook Group Event",
+    label: "Facebook Event",
     control: "MuiTextField",
     optional: true,
     group: formGroups.event,
     regEx: SimpleSchema.RegEx.Url,
-    tooltip: 'https://www.facebook.com/groups/...'
-  },
-  
-  facebookPageLink: {
```

Comment:
I decided to get rid of this field since it really doesn't make sense on posts/events. I previously added it to match fields on localGroups, but I don't think FB distinguishes between "group" events and "page" events.

Reply:
You could imagine writing a migration to move the data in facebookPageLink over to facebookLink and then deleting the field from the existing events. But sounds annoying.